WcaOnRails/config/locales/en.yml

```diff
@@ -1179,10 +1179,10 @@ en:
       #context: for a new registration
       new:
         #context: "comp_name" contains the competition's name
-        mail_subject: "Your registration for %{comp_name} is submitted and pending approval"
+        mail_subject: "Registration Pending: %{comp}"
```

Comment:
I think this won't work since the variable is %{comp_name}, not %{comp} ?

Reply:
I believe that would be correct. @dunkOnIT

Reply:
Correct! Will change this when feedback concludes on Monday (I expect the tests will not pass without it, but I'll leave this conversation unresolved as well)